integration_tests/run_pyspark_from_build.sh

```diff
@@ -223,6 +223,7 @@ else
 
     REPORT_CHARS=${REPORT_CHARS:="fE"} # default as (f)ailed, (E)rror
     STD_INPUT_PATH="$INPUT_PATH"/src/test/resources
+    DEFAULT_CONFIGS_PATH="${SCRIPTPATH}"/../dist/target/spark-rapids-default-configs.json
```

Comment:
This looks like the wrong place when running with the integration tests tarball (i.e.: CI pipelines that don't build from source).  I think we want something based on TARGET_DIR here.  The path within the tarball of this file can be under dist/target/ instead of integration_tests so this relative path still works even in the tarball.

Reply:
A couple of questions for my understanding.

> CI pipelines that don't build from source

Are these different ones from the pre-merge CI that has been running for this PR? Can you point me to where I can find about them? I'd like to know more about those CI pipelines.

> I think we want something based on TARGET_DIR here.

`TARGET_DIR` is set to `"$SCRIPTPATH"/target` in this script file. Is it overwritten to somewhere else somehow in the CI pipelines you mentioned?

> The path within the tarball of this file can be under dist/target/ instead of integration_tests so this relative path still works even in the tarball.

IIUC, you seem to be suggesting the path should be `"${TARGET_DIR}"/../../dist/target/spark-rapids-default-configs.json`? Would this work though when you run the integration test locally from the `integration_tests` directory? This is currently recommended in https://github.com/NVIDIA/spark-rapids/tree/branch-24.10/integration_tests#running.

Reply:
I talked to @gerashegalov offline and got answers 🙂

Reply:
I have updated the path as suggested.